src/bin/svg-to-ts.ts

```diff
@@ -28,17 +28,17 @@ commander
   )
   .option('-p --prefix <string>', 'prefix for the generated svg constants', DEFAULTS.prefix)
   .option('-i --interfaceName <string>', 'name for the generated interface', DEFAULTS.interfaceName)
-  .option('-s --srcDirectory <value>', 'name of the source directory', collect, [])
+  .option('-s --srcFiles <value>', 'name of the source directory', collect, [])
   .option('-o --outputDirectory <string>', 'name of the output directory', DEFAULTS.outputDirectory)
   .parse(process.argv);
 
 const { delimiter, fileName, interfaceName, outputDirectory, prefix, typeName } = commander;
 
 // Because of commander adding default value to params
 // See: https://stackoverflow.com/questions/30238654/commander-js-collect-multiple-options-always-include-default
-let srcDirectories = commander.srcDirectory;
-if (srcDirectories.length === 0) {
-  srcDirectories = DEFAULTS.sourceDirectories;
+let srcFiles = commander.srcFiles;
+if (srcFiles.length === 0) {
```

Comment:
I think somehow commander is already resolving the Regex. If I put a log statement here, I get ```./inputfiles/expressionLess.svg``` instead of ```./inputfiles/*.svg```